packages/block-editor/src/components/block-list/index.js

```diff
@@ -211,15 +202,16 @@ function Items( {
 				visibleBlocks: __unstableGetVisibleBlocks(),
 				isZoomOut: __unstableGetEditorMode() === 'zoom-out',
 				shouldRenderAppender:
+					! isSectionBlock( rootClientId ) &&
+					! getBlockEditingMode( rootClientId ) !== 'disabled' &&
```

Comment:
I notice there's one too many `!`s here, should it be this?
```suggestion
					getBlockEditingMode( rootClientId ) !== 'disabled' &&
```

or potentially this (though it makes less sense to me):
```suggestion
					getBlockEditingMode( rootClientId ) === 'disabled' &&
```